Note.java

```diff
@@ -0,0 +1,154 @@
+import java.util.Scanner;
+import java.io.PrintWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+/**
+ * Class representing a note - a file containing a body
+ * of text and that body's title.
+ * This class contains constructors and methods for creating,
+ * modifying and deleting these notes.
+ * 
+ * 
+ * @version Sprint 1, V1.0
+ */
+public class Note{
+	
+	private String titleText;
+	private String bodyText;
+	
+	private File noteFile;
+	
+	private static String notesDirectory = System.getProperty("user.home") + System.getProperty("file.separator") + "DynOSor" + System.getProperty("file.separator") + "Notes"; 
+	
+	/**
+	 * Constructor which takes a text file, with the first line representing the .
+	 * Each field name and its contents are added to their appropriate ArrayLists.
+	 * 
+	 * @param noteFile the file from which to read the note information from.
+	 */
+	public Note(File noteFile){
+		this.noteFile = noteFile;
+		
+		Scanner noteReader = null;
+		
+		try {
+			noteReader = new Scanner(this.noteFile);
+		
+			titleText = noteReader.nextLine();
+			
+			while (noteReader.hasNextLine()){
+				bodyText = bodyText + "\n" + noteReader.nextLine();
+			}
+			
+		}
+		catch (FileNotFoundException ex){
+			System.err.println("Could not read from the specified note file because the file could not be found.");
+		}
+		finally {
+			noteReader.close();
+		}
+		
+	}
+	
+	/**
+	 * Constructor which creates a new instance of Note with
+	 * the given title and body text.
+	 * 
+	 * @param title the new note's title text.
+	 * @param body the new note's body text.
+	 */
+	public Note(String title, String body){
+		titleText = title;
+		bodyText = body;
+		
+		String noteSuffix = ((Integer)(new File(notesDirectory).listFiles().length)).toString();
+		noteFile = new File(notesDirectory + "note" + noteSuffix);
```

Comment:
Need to add a check to make sure that there is no file already with this file name in the Notes directory.